scenario/utils/relayArbitrumMessage.ts

```diff
@@ -29,7 +29,13 @@ export async function relayArbitrumMessage(
     const wordLength = 2 * 32;
     const innnerData = header + data.slice(headerLength + (11 * wordLength));
     const toValue = data.slice(headerLength + (2 * wordLength), headerLength + (3 * wordLength));
-    const toAddress = BigNumber.from(`0x${toValue}`).toHexString();
+    let toAddress = BigNumber.from(`0x${toValue}`).toHexString();
+    
+    // if lenght of toAddress is less than 42, then it is padded with 0s and we need to add them after 0x
+    if(toAddress.length < 42) {
```

Comment:
Instead of 0x00936F571dDD39E5DC13291b3e93C941A208998A generated 0x936F571dDD39E5DC13291b3e93C941A208998A